pyglet/canvas/xlib.py

```diff
@@ -274,9 +274,7 @@ def restore_mode(self):
             self.set_mode(self._initial_mode)
 
     def __repr__(self):
-        return 'XlibScreen(display=%r, x=%d, y=%d, ' \
-               'width=%d, height=%d, xinerama=%d)' % \
-               (self.display, self.x, self.y, self.width, self.height, self._xinerama)
+        return f"{self.__class__.__name__}(display={self.display!r}, x={self.x}, y={self.y}, width={self.width}, height={self.height}, xinerama={self._xinerama})"
```

Comment:
Can you break the line at about 100 characters?

Reply:
got it， i will